modules/get-started/pages/quick-start.adoc

```diff
@@ -226,75 +469,249 @@ include::get-started:partial$install-rpk-homebrew.adoc[]
 --
 ======
 
+. Create a profile to configure `rpk` to connect to your cluster:
++
+[,bash]
+----
+rpk profile create quickstart --from-profile rpk-profile.yaml
+----
++
+NOTE: The profile is configured in the `docker-compose/three-brokers/rpk-profile.yaml` file that you downloaded. This file is commented to explain how it works.
+
 . Get information about the cluster to test the connection:
 +
 ```bash
-rpk cluster info -X brokers=127.0.0.1:19092
+rpk cluster info
+```
++
+NOTE: The Redpanda broker returns a list of all broker addresses, so `rpk` can communicate with all brokers directly. Each broker returns its configured `advertise-*` address that matches the port to which the initial connection has been made. You can see examples of these addresses in the Docker Compose file that you downloaded.
+
+== View offloaded data in Tiered Storage
+
+Tiered Storage keeps local disk usage stable and performance consistent by offloading older data to object storage. You can also use Tiered Storage to recover clusters or topics, mount and unmount topics from a cluster, and more.
+
+NOTE: For this quickstart, Redpanda uses MinIO as the object storage provider. MinIO is not supported for production deployments. For production, see xref:deploy:deployment-option/self-hosted/manual/production/requirements.adoc#object-storage-providers-for-tiered-storage[Object storage providers for Tiered Storage].
+
+. Check the size of remote and local disks:
++
+[,bash]
+----
+rpk cluster logdirs describe --topics logins
+----
++
+[.no-copy]
+----
+BROKER  DIR                     TOPIC   PARTITION  SIZE     ERROR
+1       /var/lib/redpanda/data  logins  0          3364032
+1       remote://redpanda       logins  0          3358193
+----
++
+- Local storage: `/var/lib/redpanda/data` holds recent data.
+- Remote storage: `remote://redpanda` holds offloaded data.
+
+. Open MinIO at http://localhost:9001/browser^ to view your data stored in the S3-compatible object store.
```

Comment:
```suggestion
. Open MinIO at http://localhost:9001/browser^ to view your data stored in the S3-compatible object store.
```
That ^ renders in the html. Do you need to do this like http://localhost:9001/browser[http://localhost:9001/browser^]? There's another external link like this above, where I just suggested adding the ^. That one will also need to be fixed similarly.